desk/app/metagrab.hoon

```diff
@@ -320,7 +333,14 @@
         ::  no valid cache entry for this target, start a new fetch
         ::
         =.  await  (~(put ju await) u.target id)
-        [[(fetch u.target) ~] this]
+        =/  uas=@t
```

Comment:
when I was testing this, it wasn't just the user agent string that made a difference

Reply:
Also the `Accept` header, right? We still include that statically as `*/*`.